9.mAP/notebooks/5.aggregate_map_scores_morphology_treatment.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<p>where there any that passed the threshold? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>